utilities/common.py

```diff
@@ -94,10 +94,11 @@ def common_parser():
     parser.add_argument("--trackerMuons", action='store_true', help="Use tracker muons instead of global muons (need appropriate scale factors too)")
     parser.add_argument("--binnedScaleFactors", action='store_true', help="Use binned scale factors (different helpers)")
     parser.add_argument("--onlyMainHistograms", action='store_true', help="Only produce some histograms, skipping (most) systematics to run faster when those are not needed")
-    parser.add_argument("-v", "--verbose", type=int, default=3, choices=[0,1,2,3,4],
-                        help="Set verbosity level with logging, the larger the more verbose (currently for setup_test_logger)");
     parser.add_argument("--met", type=str, choices=["DeepMETReso", "RawPFMET"], help="MET (DeepMETReso or RawPFMET)", default="RawPFMET")                    
     parser.add_argument("-o", "--outfolder", type=str, default="", help="Output folder")
+    parser.add_argument("--set-custom-logger", dest="setCustomLogger", action="store_true", default=False, help="Use logging with colors")
+    parser.add_argument("-v", "--verbose", type=int, default=3, choices=[0,1,2,3,4],
```

Comment:
Do these map on to ERROR, WARNING, DEBUG, INFO? If so, I'd rather have those than numbers...

Reply:
can use an enum, I guessed I took this numbering from roofit and combine, they do something similar

Reply:
Actually, it is probably simpler to keep using integers, the interpretation is easy enough and typing strings is more annoying